tools/scripts/CaptureDevHomeLogs.ps1

```diff
@@ -0,0 +1,18 @@
+# Quick script to capture log files and create a zip file for upload to GitHub
+# Currently this only creates one based on Date. We could add a time, so that 
+# the ISV can have more than one Zip file
+
+$MyZip=$(Get-Date -Format yyyy-MM-dd) + "DevHome.Zip"
+$LogPath="C:\Users\" + $Env:UserName  + "\AppData\Local\Packages"
+$LogFiles = Get-ChildItem -Path $LogPath -Filter "*.dhlog" -recurse

```

Comment:
@dkbennett do you know if DevHome holds a lock on the log files until it is closed?  If so, how would that affect this script?  I'm assuming an error will occur.

Reply:
I got a "the process cannot access the file ... because it is being used by another process" error for a dev home log in-use.

For collecting logs we probably want to add in a process termination prior to collection:

Recommend adding something like:
Get-Process *DevHome* -ErrorAction Ignore | Stop-Process

This works for me to stop dev home and our extensions.